addons/rating/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,25 @@
+class Extended_Init_EventHandlers {
+    class CAManBase {
+        class ADDON {
+            serverInit = QUOTE(_this call FUNC(initMan));
```

Comment:
Can this be added only once the ace settings are initialized on the server and either GVAR bool is false via https://github.com/CBATeam/CBA_A3/blob/master/addons/xeh/fnc_addClassEventHandler.sqf (apply retroactively set to true)?